samples/kvssink_intermittent_sample.cpp

```diff
@@ -0,0 +1,293 @@
+#include <thread>
+#include <mutex>
+#include <chrono>
+#include <condition_variable>
+
+#include <gst/gst.h>
+#include <glib.h>
+
+#include "gstreamer/gstkvssink.h"
+
+using namespace com::amazonaws::kinesis::video;
+using namespace log4cplus;
+
+/* modify these values to change start/stop interval */
+#define KVS_INTERMITTENT_PLAYING_INTERVAL_SECONDS 20
+#define KVS_INTERMITTENT_PAUSED_INTERVAL_SECONDS 40
```

Comment:
Can we add usage of this sample to the readme + also explain high level what it does (streams to kvs for 20 seconds then stops streaming for 40 seconds, and repeats the cycle indefinitely)